polyhost/services/unicode_cache.py

```
@@ -18,19 +18,26 @@ def __init__(self, size: int = 32):
         url = f"https://cdn.jsdelivr.net/gh/twitter/twemoji@14.0.2/assets/72x72/{codepoints}.png"
 
         try:
-            response = requests.get(url)
+            response = requests.get(url, timeout=5)
             response.raise_for_status()
             pixmap = QPixmap()
             pixmap.loadFromData(response.content)
             pixmap = pixmap.scaled(self.size, self.size)
             pixmap.save(str(filename))
             print(f"[UnicodeCache] Cached: {filename.name}")
         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)